addon/components/frost-object-browser.js

```diff
@@ -50,6 +51,16 @@ export default Ember.Component.extend({
       return {}
     }
   },
+  
+  /**
+   * Verifies the state of showCountInSubTitle variable before showing the subtitle
+   * (in case someone doesn't need the counts)
+   */
+  @readOnly
+  @computed('subtitle', 'showCountInSubTitle', 'computedValuesTotal')
+  computedSubtitle (subtitle, showCountInSubTitle, count) {
```

Comment:
With these ES7 decorators for computer properties the JSDoc actually needs to go after `@computed` in order for the linter to catch errors such as missing parameters.
```javascript
  @readOnly
  @computed('subtitle', 'showCountInSubTitle', 'computedValuesTotal')
  /**
   * Verifies the state of showCountInSubTitle variable before showing the subtitle
   * (in case someone doesn't need the counts)
   */
  computedSubtitle (subtitle, showCountInSubTitle, count) {
```
After you make the above change the linter is going to complain your JSDoc is missing all three function parameters as well as the return and thus you should end up with something like:
```javascript
  @readOnly
  @computed('subtitle', 'showCountInSubTitle', 'computedValuesTotal')
  /**
   * Verifies the state of showCountInSubTitle variable before showing the subtitle
   * (in case someone doesn't need the counts)
   * @param {String} subtitle - subtitle text
   * @param {Boolean} showCountInSubTitle - whether or not to show count in subtitle
   * @param {Number} count - number of items in list
   * @returns {String} full subtitle to show in UI
   */
  computedSubtitle (subtitle, showCountInSubTitle, count) {
```